packages/api-nest/test/helpers/create-e2e-app.ts

```
@@ -0,0 +1,159 @@
+ *
+ * @returns Module fixture and initialized application
+ *
+ * @example
+ * ```typescript
+ * const { app, moduleFixture } = await createE2EAppWithModule()
+ *
+ * const userRepository = moduleFixture.get<Repository<User>>(
+ *   getRepositoryToken(User)
+ * )
+ * ```
+ */
```

ESLint

[error] Missing JSDoc return type.

(valid-jsdoc)